services/frontend/src/components/profileForms/careerManagementForm/CareerManagementFormView.scss

```diff
@@ -4,14 +4,9 @@
   min-height: 343px;
   padding: 30px 30px;
 }
-.pgf-formTitle {
```

Comment:
@dynamic11 The ".pgf-formTitle" class is being used at 2 others places in CareerManagementFormView.jsx, but it was removed from the CareerManagementFormView.scss. Should we refactor this?

See screenshot

![Screen Shot 2020-11-13 at 2 24 13 PM](https://user-images.githubusercontent.com/29315512/99112627-f685ad80-25bb-11eb-8ad3-4cf3415b34f0.png)

Reply:
Yes! I will look into it